entrypoint.sh

```diff
@@ -1,3 +1 @@
-#!/bin/bash
-
-poetry run gunicorn --bind "0.0.0.0:${PORT:-5000}" "todo_app.app:create_app()"
\ No newline at end of file
+kubectl port-forward service/module-14 7080:80
```

Comment:
Even if you kept the entrypoint, your kubernetes deployment still wouldn't work. You set this in deployment.yml:

```
        ports:
        - containerPort: 80
```

But your todo app doesn't listen to port 80.